src/renderer.js

```diff
@@ -43,49 +43,58 @@ const openBrowser = appName =>
     )
 
 // Listen for installedBrowsers
-electron.ipcRenderer.on('installedBrowsers', (event, installedBrowsers) => {
-  document.getElementById('loading').style.display = 'none'
-  installedBrowsers
-    .map(browser => {
-      // use alias as label if available, otherwise use name
-      if (!browser.alias) {
-        browser.alias = browser.name
-      }
-      return browser
-    })
-    .sort((a, b) => {
-      // alphabetise
-      if (a.alias < b.alias) return -1
-      if (a.alias > b.alias) return 1
-      return 0
-    })
-    .map(browser => {
-      const listItem = document.createElement('li')
+electron.ipcRenderer.on(
+  'installedBrowsers',
+  (event, installedBrowsers, notifications) => {
+    const listKeys = []
```

Comment:
what does this do?

Reply:
If you are referring to ```const listKeys = []```, it's part of the logic that handle duplicated keys, allowing only the first occurrence to be set.